sdk/search/Azure.Search.Documents/CHANGELOG.md

```diff
@@ -1,14 +1,12 @@
 # Release History
 
-## 11.4.0-beta.2 (Unreleased)
+## 11.4.0-beta.2 (2021-08-10)
 
 ### Features Added
-
-### Breaking Changes
+- Support for [Azure Active Directory](https://docs.microsoft.com/azure/active-directory/authentication/) based authentication. Users can specify a [`TokenCredential`](https://docs.microsoft.com/dotnet/api/azure.core.tokencredential) when creating a `SearchClient`, `SearchIndexClient` or a `SearchIndexerClient`. The tokens use the "https://search.azure.com/.default" scope.
```

Comment:
I don't know that the scope matters here because it's not configurable, but could we point folks to a sample?  I think this one's going to be a big deal for customers.

Reply:
I thought of mentioning the scope because it is not configurable. That way customers know that this part is not very flexible currently.

Reply:
The service is never changing that value so it won't ever be configurable.  It just feels like more of an implementation detail than something that belongs in the changelog.  For the changelog I think we'd want a line more like: 

> You can get started with `new SearchClient(endpoint, new DefaultAzureCredential())` to authenticate via AAD using [Azure.Identity](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/identity/Azure.Identity/README.md).

but even then it's probably better to link to a sample that also talks about setting up RBAC for your Search resources.  I suppose it'd be fine to put the auth scope as a remark on the .ctors taking a TokenCredential for comprehensiveness, but I don't know what any customers would do with that knowledge.

Reply:
I'll make the change. I thought I came across issues filed in other SDKs where users wanted scopes of different granularity to be used. I am not sure now. So, I'll remove it for now to keep the doc simple.

Reply:
Yes, but (unfortunately) it's service by service.  Some have a single global scope, some have different scopes by cloud that are automatically discoverable via a well known endpoint, and some have different scopes that customers need to configure in their application.  We'd only ever expose it to customers in that last case.  Search is currently going with the single global approach from the last conversation we had about it.

Reply:
Ok, that's what I was remembering - 'Storage' where we have 'different scopes that customers need to configure in their application'.
Thanks for reminding where we are in 'Search'.